workflow-language/src/main/java/com/symphony/bdk/workflow/swadl/v1/activity/Obo.java

```diff
@@ -0,0 +1,13 @@
+package com.symphony.bdk.workflow.swadl.v1.activity;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import javax.annotation.Nullable;
+
+@EqualsAndHashCode
+@Data
+public class Obo {
+  @Nullable private String username;
```

Comment:
should we use camelcase ?

Reply:
`username` is a single word in all (non-generated) places in bdk and wdk

Reply:
You're right, username is used as a single world. 
FYI, we use kebab case for the naming convention. user id  for example, is written `user-id` in the json-schema, and mapped to `userId`. This means if one day we want to have `userName` in the pojos, we will have to update the property in SWADL to `user-name`.